src/App.jsx

```diff
@@ -1,33 +1,45 @@
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
+export const App = () => {
+  const [query, setQuery] = useState('');
+  const Text = text => text.toLowerCase().trim();
```

Comment:
it is not a constructor or a component, so the function should be named with a lowercase letter
```suggestion
  const queryText = text => text.toLowerCase().trim();
```

OR 

```suggestion
  const processText = text => text.toLowerCase().trim();
```